cost-analyzer/templates/cost-analyzer-oidc-config-map-template.yaml

```diff
@@ -1,5 +1,3 @@
-{{- if .Values.oidc }}
```

Comment:
Why remove this?

Reply:
Apologies. a remnant of old hackathon testing. Re-added.

Thanks for the catch.